build/bot/Dockerfile

```diff
@@ -15,5 +15,5 @@ COPY package.json yarn.lock ./
 RUN yarn install
 
 COPY src/ .
-
+COPY conf/ /configuration/
```

Comment:
We decided not to build the container with all links attached. Let's use volume bindings as before

Reply:
Resolved: container pushed to private repository